docs/differences-from-v1.md

```diff
@@ -81,3 +221,29 @@ If you are integrating a private project and want to share more details, or if y
 In 1.x, `fossa report` supported `fossa report dependencies`, `fossa report licenses` and `fossa report attribution`.
 
 In 2.x, only `fossa report attribution --json` is reported. The information in the previously supported v1 commands is contained in the output from `attribution`.
+
+## Frequently Asked Questions
+
+#### Until when CLI v1 will be supported by FOSSA?
+
+FOSSA 1.x CLI is available and can be used indefinitely. 
+
+FOSSA will only patch 1.x for security fixes. Any bug and patch development work will occur in 2.x moving forth. 
+
+#### I'm getting a poor result with CLI v2 compared to CLI v1.
```

Comment:
This should probably be CLI greater than version 2? I'm not sure the best way to say this, but we need to figure out a way to clearly state this with the upgrade.